fe/fe-core/src/main/java/com/starrocks/connector/ConnectorPartitionTraits.java

```diff
@@ -435,6 +447,56 @@ public Optional<Long> maxPartitionRefreshTs() {
             IcebergTable icebergTable = (IcebergTable) table;
             return icebergTable.getSnapshot().map(Snapshot::timestampMillis);
         }
+
+        @Override
+        public PartitionKey createPartitionKey(List<String> partitionValues, List<Column> partitionColumns)
+                throws AnalysisException {
+            Preconditions.checkState(partitionValues.size() == partitionColumns.size(),
+                    "columns size is %s, but values size is %s", partitionColumns.size(),
+                    partitionValues.size());
+
+            IcebergTable icebergTable = (IcebergTable) table;
+            List<PartitionField> partitionFields = Lists.newArrayList();
+            for (Column column : partitionColumns) {
```

Comment:
Do you consider partition evolution？ Does it can work normally?

Reply:
already disable